app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1 @@
+$grid-float-breakpoint: 0px;
```

Comment:
Files should end with a trailing newline<br>`0px` should be written without units as `0`